# Vectorized array that should unpack to size (num_rows, num_cols)
float64[] data
int64     num_rows
int64     num_cols